repology/parsers/parsers/sisyphus.py

```diff
@@ -0,0 +1,38 @@
+from typing import Iterable
+
+from repology.packagemaker import PackageFactory, PackageMaker, LinkType
+from repology.parsers import Parser
+from repology.parsers.json import iter_json_list
+
+
+class SisyphusJsonParser(Parser):
+    def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMaker]:
+        for packagedata in iter_json_list(path, ('packages', None)):
+            with factory.begin() as pkg:
+                pkg.add_name(packagedata['name'])
+                pkg.set_version(packagedata['version'])
+                pkg.set_rawversion(
+                    f"{packagedata['epoch']}:{packagedata['version']}-{packagedata['release']}"
+                )
+                pkg.add_categories(packagedata['category'])
+                pkg.add_homepages(packagedata['url'])
+                pkg.set_summary(packagedata['summary'])
+                pkg.add_licenses(packagedata['license'])
+                pkg.add_maintainers(packagedata['packager'])
+                # store package epoch and release
+                pkg.set_extra_field('epoch', packagedata['epoch'])
+                pkg.set_extra_field('release', packagedata['release'])
```

Comment:
This is not needed. `extrafields` are only used as a way to pass some extra data to packaglink templates, and versions are not used there.

Reply:
done